sql/donut.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS members;
+CREATE TABLE members (
+    member_id int(11) NOT NULL AUTO_INCREMENT, -- Primary Key
+    uid char(7) NOT NULL,  -- Unique
+    last_name varchar(255) NOT NULL, 
+    first_name varchar(255) NOT NULL,
+    middle_name varchar(255), 
+    email varchar(255) NOT NULL,
+    phone varchar(64) DEFAULT NULL,
+    gender enum('m', 'f') DEFAULT NULL, -- Question: are there PC options we need to add?
+    birthday date DEFAULT NULL,
+    entry_year year(4) DEFAULT NULL,
+    graduation_year year(4) DEFAULT NULL, -- Question: does this 1-to-1 denote year of school (frosh, soph)
+    msc smallint(6) DEFAULT NULL,
+    building varchar(32) DEFAULT NULL,
+    room_num smallint(6) DEFAULT NULL,
+    is_abroad bool DEFAULT 0,
+    major varchar(16) DEFAULT NULL,
+    major2 varchar(16) DEFAULT NULL,
+    address varchar(255) DEFAULT NULL, -- Someone confirm how big this varchar should be.
+    city varchar(64) DEFAULT NULL,
+    state varchar(64) DEFAULT NULL,
+    zip varchar(9) DEFAULT NULL,
+    country varchar(64) DEFAULT NULL, 
+    create_account_key char(32) DEFAULT NULL,
+    PRIMARY KEY (member_id),
+    UNIQUE KEY uid (uid)
+);
+
+DROP TABLE IF EXISTS users;
```

Comment:
If you tried to run the script twice in succession right now, it would fail because `users` has a FK to `members`, so your `DROP TABLE IF EXISTS members` statement would fail (well, it might not fail if the users table is empty, but if there *were* rows then it would fail).